changelogs/unreleased/4209-module-install-pip-check.yml

```
@@ -0,0 +1,5 @@
+---
+description: run "pip check" after module installations to ensure no requirements have been broken.
+issue-nr: 4209
```

This issue number isn't correct. And the one in the PR references itself.
```suggestion
issue-nr: 3383
```